contracts/HubPool.sol

```diff
@@ -339,19 +459,35 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
         return _liquidityUtilizationPostRelay(l1Token, relayedAmount);
     }
 
-    function sync(address l1Token) public nonReentrant {
+    /**
+     * @notice Synchronize any balance changes in this contract with the utilized & liquid reserves. This should be done
+     * at the conclusion of an L2 -> L1 token transfer via the canonical token bridge, for example, when this contract's
+     * reserves do not reflect all of its token balance.
+     */
+    function sync(address l1Token) public override nonReentrant {
         _sync(l1Token);
     }
 
     /*************************************************
      *             DATA WORKER FUNCTIONS             *
      *************************************************/
 
-    // After proposeRootBundle is called, if the any props are wrong then this proposal can be challenged. Once the
-    // challenge period passes, then the roots are no longer disputable, and only executeRootBundle can be called and
-    // this can't be called again until all leafs are executed.
-    // Note: bundleEvaluationBlockNumbers must contain the latest block number for _all_ chains, even if there are no
-    // relays contained on some of them.
+    /**
+     * @notice Publish a new root bundle to this contract along with all of the block numbers that the merkle roots
+     * are relevant for. This is used to aid off-chain validators in evaluating the correctness of this bundle. Caller
+     * must stake a bond that can be slashed if the root bundle proposal is invalid, and they will receive it back
+     * if accepted.
```

Comment:
sorry for doing this but I really prefer if we dont wrap like this. it waists a fair bit of space. I think my recommendation that I made will look correct but you might need to validate it in your editor.

```suggestion
     * @notice Publish a new root bundle to along with all of the block numbers that the merkle roots are relevant for. 
     * This is used to aid off-chain validators in evaluating the correctness of this bundle. Caller stakes a bond that can
     * be slashed if the root bundle proposal is invalid, and they will receive it back if accepted.
```